IronFoundry.Container.Host/App.config

```diff
@@ -1,5 +1,23 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <configuration>
+  <configSections>
+    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+  </configSections>
+
+  <!-- <nlog throwExceptions="true" internalLogToConsole="true" internalLogLevel="Debug" -->
+  <nlog throwExceptions="true" async="false" xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <targets>
+      <target name="console" xsi:type="Console" layout="${longdate}|${level:uppercase=true}|${threadid}|${logger:shortName=true}|${message}${onexception:|${exception:format=message,stacktrace:maxInnerExceptionLevel=10:innerFormat=message,stacktrace}}" />
+
+      <target name="file" xsi:type="File" layout="${longdate}|${level:uppercase=true}|${threadid}|${logger:shortName=true}|${message}${onexception:|${exception:format=message,stacktrace:maxInnerExceptionLevel=10:innerFormat=message,stacktrace}}" fileName="logs\hostlog.txt" archiveFileName="logs\hostlog-{#}.txt" archiveEvery="Day" archiveNumbering="Rolling" maxArchiveFiles="7" concurrentWrites="true" keepFileOpen="false" />
```

Comment:
This is fine for now (for debugging purposes), but ultimately the container user should not have access to the logs.  We'll create a card to track changing this.